hvac/api/system_backend/namespace.py

```diff
@@ -47,3 +48,54 @@ def delete_namespace(self, path):
             url=api_path,
         )
         return response
+
+    def list_policies_namespace(self, path):
```

Comment:
I'm not familiar with this API route. Do you know if its recorded under https://www.vaultproject.io/api/system/ or otherwise? It feels like it may fit better under https://github.com/hvac/hvac/blob/develop/hvac/api/system_backend/policy.py but I'd want to look up the upstream code / documentation to get a better idea of that scope before saying for sure...

Reply:
https://learn.hashicorp.com/vault/operations/namespaces#api-call-using-curl-1

Above document explains how this will work

Reply:
I am referring to below API call in the document
API call using cURL
To target a specific namespace, you can do one of the following:

Option 1: Pass the target namespace in the X-Vault-Namespace header

Option 2: Prepend the API endpoint with namespace name (e.g. <namespace_name>/sys/policies/acl)